docs/ocean/design_docs/prognostic-sediment-transport.rst

```diff
@@ -0,0 +1,476 @@
+
+Prognostic non-cohesive sediment transport model
+================================================
+
+date: 2021/01/14
+
+Contributors: Zhendong Cao
+
+
+
+Summary
+-------
+
+The current MPAS-Ocean includes a sediment transport analysis member. But it lacks
+the capabilities to simulate the morphological change and the potential interactions
+between sediment transport and other system components (e.g., flow field, vegetation dynamics, etc.).
+Therefore, we propose a prognostic sediment transport model to enable these missing capabilities.
+
+The proposed model development will focus on the non-cohesive sediment transport.
+The model requirements, algorithms and implementation mainly follow the sediment transport
+development work in `Warner et al., 2005 <https://doi.org/10.1016/j.cageo.2008.02.012>`_ and
+the `COAWST code (USGS version) <https://github.com/jcwarner-usgs/COAWST>`_.
+The main model components include
+1) sediment classes;
+2) bedload sediment transport;
+3) suspended sediment transport; and
+4) morphological change.
```

Comment:
If you want this to come out formatted like a numbered list, you need blank lines:
```suggestion

1) sediment classes;

2) bedload sediment transport;

3) suspended sediment transport; and

4) morphological change.

```